### PR TITLE
added characters card

### PR DESCRIPTION
**Pull request by Aijazmakerb**
added chracters section area in details page, hope you will like it

### CODE REVIEW

**Comment by DevanAbinaya** [comment]
Hi! Thanks for the PR. I'll take a look at it later this afternoon. :)

File: components/anime/charactersCard.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            <div className="flex items-center justify-between lg:gap-3 z-40 px-3">
+                <h1 className="font-karla text-[20px] font-bold">Characters</h1>
+                {info?.length > 6 && (
+                    <div className="cursor-pointer font-karla" onClick={() => setShowAll(!showAll)}>
+                        {showAll ? "show less" : "show more"}
+                    </div>
+                )}
+            </div>
+            <div className="grid w-full grid-cols-1 gap-[12px] md:gap-4 md:grid-cols-3 md:pt-7 md:pb-5 px-3 md:px-5 pt-4">
+                {info.slice(0, showAll ? info.length : 6).map((item, index) => {
```
Comment (DevanAbinaya) [inline components/anime/charactersCard.js:20]:
```suggestion
                {info?.slice(0, showAll ? info.length : 6).map((item, index) => {
```

File: pages/en/anime/[...id].js [hunk 2]
```diff
@@ -185,6 +186,12 @@ export default function Info({ info, color }) {
             setWatch={setWatch}
           />
 
+          {info && (
```
Comment (DevanAbinaya) [inline pages/en/anime/[...id].js:189]:
```suggestion
          {info?.characters?.edges && (
```

**Review by DevanAbinaya — CHANGES_REQUESTED** [review]
looks good to me!
Just need a slight change to the code, after that I'll start merging.

**Comment by Aijazmakerb** [comment]
Thanks for the suggestions.

**Comment by DevanAbinaya** [comment]
and also once more, 
could you update release.md and package.json version?

release.md:
```
# Changelog

This document contains a summary of all significant changes made to this release.

## 🎉 Update v4.0.5

### Added

- Added characters card to info page
```

package.json:
```
"version": "4.0.5",
```

**Review by DevanAbinaya — APPROVED** [review]
great!

**Comment by DevanAbinaya** [comment]
great, I will start merging, thanks for your contributions. If you have any suggestion or idea you want to contribute, feel free to open pull request :)

**Pull request merged**